environment/src/test/java/com/sequenceiq/environment/environment/service/cleanup/EnvironmentStructuredEventCleanupServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.environment.environment.service.cleanup;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.sequenceiq.cloudbreak.structuredevent.service.db.CDPStructuredEventDBService;
+import com.sequenceiq.cloudbreak.util.TimeUtil;
+
+@ExtendWith(MockitoExtension.class)
+class EnvironmentStructuredEventCleanupServiceTest {
+
+    @Mock
+    private TimeUtil mockTimeUtil;
+
+    @Mock
+    private CDPStructuredEventDBService mockCDPStructuredEventService;
+
+    private EnvironmentStructuredEventCleanupService underTest;
+
+    @BeforeEach
+    void setUp() {
+        underTest = new EnvironmentStructuredEventCleanupService(mockCDPStructuredEventService, mockTimeUtil);
```

Comment:
Why not leave this to `@InjectMocks`? I do not think `setUp()` is necessary here.

Reply:
After private discussion, this can be left as is.